models/bluerov2/model.sdf.in

```diff
@@ -21,25 +21,18 @@
         </inertia>
       </inertial>
 
-      <visual name="base_link_box">
-        <pose>0 0 0 0 0 0</pose>
-        <transparency>0.5</transparency>
+      <visual name="base_link_visual">
+        <pose>-0.134 -0.4935 0.17 @d90 0 @d90</pose>
```

Comment:
Why isn't this (0, 0, 0)?

Reply:
Nice catch! I didn't adjust the origin of the CAD file when generating the STL. I centered the geometry to (0, 0, 0) with the updated COLLADA files.